paraclear/get_balance_onchain.py

```diff
@@ -0,0 +1,240 @@
+import asyncio
+import re
+import requests
+import os
+
+from dotenv import load_dotenv
+from decimal import Decimal
+from enum import Enum
+from rich import print
+from typing import Callable, Optional, Union
+
+from starknet_py.cairo.felt import FIELD_PRIME
+from starknet_py.constants import RPC_CONTRACT_ERROR
+from starknet_py.contract import Contract
+from starknet_py.hash.selector import get_selector_from_name
+from starknet_py.net.client import Client
+from starknet_py.net.client_errors import ClientError
+from starknet_py.net.client_models import Call
+from starknet_py.net.full_node_client import FullNodeClient
+from starknet_py.net.models import Address
+from starknet_py.proxy.contract_abi_resolver import ProxyConfig
+from starknet_py.proxy.proxy_check import ArgentProxyCheck, OpenZeppelinProxyCheck, ProxyCheck
+
+FEE_TOKEN_QUANTUMS = 18
+PARACLEAR_QUANTUMS = 8
+USDC_QUANTUMS = 6
+
+print("Current working directory:", os.getcwd())
+print("Trying to load .env file...")
+load_dotenv()
+print(".env file loaded (if it exists)")
+
+# Debug: Print all environment variables
+for key, value in os.environ.items():
+    print(f"{key}: {value}")
```

Comment:
Do we need this for customers?  For me this doesn't inspire confidence that this will work if we have debug info in by default.